### PR TITLE
4588 - Removed summary widget creatability and updated composition policy

### LINKED ISSUE #4588: Remove Angular code from Summary Widgets and make them non-creatable
#### Summary
* Replace [the legacy composition policy](https://github.com/nasa/openmct/blob/master/src/plugins/summaryWidget/plugin.js#L88) with [a new-style composition policy](https://github.com/nasa/openmct/blob/master/src/plugins/condition/plugin.js#L62).
* [Remove the summary widget view policy]( https://github.com/nasa/openmct/blob/master/src/plugins/summaryWidget/plugin.js#L93). I think it's just suppressing telemetry views like plots and tables for summary widgets, which themselves produce telemetry (the output of the conditional evaluation). I am fine if those views are available for now.
* [Remove the call to the objectService](https://github.com/nasa/openmct/blob/master/src/plugins/summaryWidget/src/SummaryWidget.js#L108) (not even sure what it's doing).
* Remove [the `creatable` property from the Summary Widget Object type](https://github.com/nasa/openmct/blob/master/src/plugins/summaryWidget/plugin.js#L20).

### PR DESCRIPTION
**Pull request by michaelrogers**
Closes #4588 

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [ ] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [ ] Unit tests included and/or updated with changes?
* [ ] Command line build passes?
* [ ] Has this been smoke tested?
* [x] Testing instructions included in associated issue?

How to test:
1. Open the create menu
2. Ensure that `Summary Widget` is not available in the menu

### CODE REVIEW

File: src/plugins/summaryWidget/plugin.js [hunk 1]
```diff
@@ -85,16 +84,7 @@ define([
...
-                category: 'composition',
-                implementation: SummaryWidgetsCompositionPolicy,
-                depends: ['openmct']
-            });
-            openmct.legacyExtension('policies', {
-                category: 'view',
-                implementation: SummaryWidgetViewPolicy,
-                depends: ['openmct']
-            });
+            openmct.composition.addPolicy(new SummaryWidgetsCompositionPolicy(openmct).allow);
```
Comment (akhenry) [inline src/plugins/summaryWidget/plugin.js:87]: Is `new` needed here?

Comment (akhenry) [reply src/plugins/summaryWidget/plugin.js:87]: Ah, I was reading it wrong.

I think what you'll need is:
```
let compositionPolicy = new SummaryWidgetsCompositionPolicy(openmct));
openmct.composition.addPolicy(compositionPolicy.allow.bind(compositionPolicy));
```

Comment (michaelrogers) [reply src/plugins/summaryWidget/plugin.js:87]: I'll update that, thank you.

Comment (michaelrogers) [reply src/plugins/summaryWidget/plugin.js:87]: Should the condition widget be updated to be consistent with that approach? https://github.com/nasa/openmct/blob/master/src/plugins/condition/plugin.js#L62

Comment (akhenry) [reply src/plugins/summaryWidget/plugin.js:87]: Probably a good idea. I don't really understand why it's been done that way.

File: src/plugins/condition/plugin.js [hunk 2]
```diff
@@ -54,7 +54,12 @@ export default function ConditionPlugin() {
                 domainObject.telemetry = {};
             }
         });
-        openmct.composition.addPolicy(new ConditionSetCompositionPolicy(openmct).allow);
+        openmct.legacyExtension('policies', {
```
Comment (akhenry) [inline src/plugins/condition/plugin.js:57]: This might be a merge error, this has been removed in master.

Comment (michaelrogers) [reply src/plugins/condition/plugin.js]: Agreed, it was a merge error. I've pushed an update to remove the legacyExtension.

**Comment by akhenry** [comment]
Looks good to me, thanks!

**Review by akhenry — APPROVED** [review]
### Reviewer Checklist

1. Changes appear to address issue? Y
2. Appropriate unit tests included? N/A
3. Code style and in-line documentation are appropriate? Y
4. Commit messages meet standards? Y
5. Has associated issue been labelled `unverified`? (only applicable if this PR closes the issue) Y

**Pull request merged**